tests/test-dirs/config/flags/invalid.t

```diff
@@ -2,7 +2,9 @@
   {
     "class": "return",
     "value": {
-      "dot_merlins": [],
+      "dot_merlins": [
+        "tests/test-dirs/config/flags/.merlin"
+      ],
```

Comment:
Why did you add a `.merlin` file here?

Reply:
To prevent triggering the dune reader and having a path not correctly sanitized by the wrapper:

`"/home/ulysse/tmp/merlin/dune-project"` 

I could also have fixed that by adding a sanitization rule to the merlin wrapper.

Reply:
Oh right, that makes sense. Thanks.
It's fine like this.